.circleci/config.yml

```diff
@@ -50,13 +50,22 @@ jobs:
           RAILS_ENV: test
           PULMAP_DB_HOST: localhost
           PULMAP_DB_USERNAME: geoblacklight
+      - image: solr:7.1-alpine
+        command: bin/solr -cloud -noprompt -f -p 8985
     steps:
       - attach_workspace:
           at: '~/pulmap'
       # Bundle install dependencies
       - run: bundle install --path vendor/bundle
+      - run:
+          name: Load config into solr
+          command: |
+            cd solr/conf
+            zip -1 -r solr_config.zip ./*
+            curl -H "Content-type:application/octet-stream" --data-binary @solr_config.zip "http://localhost:8985/solr/admin/configs?action=UPLOAD&name=pulmap"
+            curl -H 'Content-type: application/json' http://localhost:8985/api/collections/ -d '{create: {name: pulmap-core-test, config: pulmap, numShards: 1}}'
       - run: bundle exec rake db:migrate
-      - run: bundle exec rake ci
+      - run: bundle exec rspec spec/
```

Comment:
Do you need to add a `jetty_wait` script like Figgy has?  https://github.com/pulibrary/figgy/blob/master/bin/jetty_wait

Reply:
I don't think we use `jetty_wait` anymore in Figgy. We use it in Orangelight, but I think I like the new method better. Appears to work now.